server/etcdmain/help.go

```diff
@@ -78,7 +78,7 @@ Member:
   --memory-mlock
     Enable to enforce etcd pages (in particular bbolt) to stay in RAM.
   --quota-backend-bytes '0'
-    Raise alarms when backend size exceeds the given quota (0 defaults to low space quota).
+    Sets the maximum size (in bytes) that the etcd backend database may consume. If the quota is exceeded, etcd raises an alarm and transitions into read-only mode, preventing further writes. A value of 0 applies the default low space quota of 2 GiB.
```

Comment:
> Sets the maximum size (in bytes) that the etcd backend database may consume

I would be careful about saying that it sets a maximum. This is not a accurate hard max, it's based on heuristic executed that etcd tries to guess that write will go over the maximum. The real database size doesn't grow linearly, but is executed as we run out of free pages. I expect that bbolt does some overallocations for future, but I haven't read code recently to say how much.

cc @ahrtr

Reply:
> This is not a accurate hard max, it's based on heuristic executed that etcd tries to guess that write will go over the maximum.

IMO, this is an implementation detail, which is not of relevance here. As the one who opened the original issue, I find this explanation sufficiently accurate. Thanks @zechariahks for taking care of this!

To the maintainers: Pls. ensure this documentation update will be properly reflected in the v3.5 and v3.6 docs.

Reply:
Thank you @derselbst @ahrtr for the detailed explanation. I just added the sign off to the commit.